.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,363 @@
+Affero
+Aissue
+Akechi
+Aopen
+Args
+Bienvenido
+CDNs
+CQA
+DOCTYPE
+Eqt
+Everyone
+FReeshabh
+GSo
+Giridhar
+Google
+HMAC
+Habilidad
+Hacktoberfest
+Harmandeep
+Hrmn
+INI
+Kamdar
+MYUSERNAME
+Manjaro
+Mystylesheet
+Niraj
+OCSP
+Prajwal
+Prasath
+Purvanshsingh
+Qqe
+Rahul
+SPDX
+SYSV
+Saurabh
+TODO
+VPkg
+Vorbis
+Xiph
+YYYY
+abhaykatheria
+ableabhinav
+accountsservice
+adriennefriend
+amd
+ananthan
+anthonyharrison
+api
+apk
+asn
+autoescape
+autoextract
+autoextracts
+backend
+backends
+backport
+backported
+bcca
+bdbd
+berenjena
+bigbird
+binutils
+bksahu
+blog
+bluetooth
+bluetoothctl
+bluez
+brainwane
+brotlipy
+btn
+bubblewrap
+bugfixes
+busybox
+bzip
+cabextract
+cfea
+cff
+chaitanyamogal
+charset
+checkername
+chris
+cleartext
+clnt
+cmdline
+codecov
+config
+copyleft
+cpio
+cpp
+crond
+cryptsetup
+csrc
+css
+csv
+customisation
+customise
+customising
+cve
+cvedb
+cvedetails
+cves
+cvss
+cyberciti
+cygwin
+dbus
+debian
+debuginfo
+delimted
```

Comment:
This looks like a typo.

Reply:
Fixed it and also double checked the allow.txt for any other typos